### DISCUSSION TITLE
SQLPad

### DISCUSSION THREAD

**Discussion by amotl** [crate/crate-clients-tools#204] [Ideas]
Hi there,

we might want to investigate whether this works and then add [SQLPad](https://sqlpad.github.io/sqlpad/) to the list.

With kind regards,
Andreas.

cc @proddata

**Comment by proddata** [comment]
should work ... in fact I reworked the ui a bit for crate https://github.com/sqlpad/sqlpad/pull/793 😬

**Comment by amotl** [comment]
Thanks for reworking the UI.

Unfortunately, SQLPad is in maintenance mode now. Alternatives are listed on https://getsqlpad.com/en/introduction/#alternatives. I will add them to the [backlog](../blob/main/BACKLOG.md) file.

- [Querybook](https://www.querybook.org/) (open source)
- [Superset](https://github.com/apache/superset) (open source) / [Preset](https://preset.io/) (company)
- [Chartbrew](https://github.com/chartbrew/chartbrew) (open source) / [Chartbrew](https://chartbrew.com/) (company)
- [Lightdash](https://github.com/lightdash/lightdash) (open source) / [Lightdash](https://www.lightdash.com/) (company)
- [CloudBeaver](https://github.com/dbeaver/cloudbeaver) (open source)
- [Awesome DB tools GUIs](https://github.com/mgramin/awesome-db-tools#gui)
